client/app/visualizations/counter/Editor/PrimaryValueSettings.jsx

```diff
@@ -0,0 +1,22 @@
+import { omit } from "lodash";
+import React, { useCallback } from "react";
+import { EditorPropTypes } from "@/visualizations/prop-types";
+import CounterValueOptions from "./CounterValueOptions";
+import counterTypes from "../counterTypes";
+
+export default function PrimaryValueSettings({ options, data, onOptionsChange }) {
+  const onChange = useCallback(primaryValue => onOptionsChange({ primaryValue }), [onOptionsChange]);
```

Comment:
Since nothing really depends on `onChange` for rendering condition, I think we can remove this `useCallback` and just have that method directly on the `onChange={}` below.

Reply:
Done (and the second one as well)